src/platform/device.gni

```diff
@@ -23,9 +23,6 @@ declare_args() {
 
   # Substitute fake platform when building with chip_device_platform=auto.
   chip_fake_platform = false
-
-  # Include wifi-paf to commission the device or not
-  chip_device_config_enable_wifipaf = false
```

Comment:
Let's add this back in

Reply:
May we change it as: 
chip_device_config_enable_wifipaf = false && chip_enable_wifi
Since it can be enabled only while wifi is enabled.